src/cam.ts

```diff
@@ -75,24 +77,49 @@ export class CameraController {
     this.render()
   }
 
-  private render() {
-    const transform: string[] = []
-    const classList: string[] = []
+  public clip() {
+    this.isClipped = !this.isClipped
 
-    transform.push(`translate(${this.position.x}%, ${this.position.y}%)`)
-    transform.push(`scale(${this.position.z})`)
+    this.render()
+  }
+
+  private applyPositioning() {
+    this.videoElement.style.transform = `translate(${this.position.x}%, ${this.position.y}%) scale(${this.position.z})`
 
     if (this.isFlipped) {
-      classList.push('flip')
-      transform.push('rotateY(180deg)')
+      this.videoElement.style.transform += 'rotateY(180deg)'
```

Comment:
The only part where I prefer the way it was working is here. 

I prefer storing what classes and properties we gonna add to the videoElement and wrapperElement and after all calculations we apply then only once instead of adding classes or styles serveral times during the calculations.

Do you think this is a better approach than the one we applied so far?

Reply:
I do, because, in this case, is just a matter of concatenating string, not making array operations.